packages/elements/src/components/ino-accordion/ino-accordion.scss

```
@@ -1,38 +1,97 @@
 @use 'base/typography-new' as typography;
 @use 'base/new-theme' as theme;
 
+
 .ino-accordion {
   &:hover {
-    background: #f6fafe;
+    background: theme.$p-01;
   }
   border-radius: 24px;
 
   &--header {
```

Should be `__header` because it's another block inside the component. `--` should be used to indicate a state of a block (e.g. `ino-accordion__title--selected`).

We are not strictly following BEM but you can read more about it here: https://css-tricks.com/bem-101/

```suggestion
  &__header {
```